src/caduceus/http.go

```diff
@@ -35,16 +37,16 @@ func (sh *ServerHandler) ServeHTTP(response http.ResponseWriter, request *http.R
 		response.WriteHeader(http.StatusBadRequest)
 		response.Write([]byte(fmt.Sprintf("Unsupported method \"%s\"... Caduceus only supports \"POST\" method.\n", request.Method)))
 
-		debugLog.Log(messageKey, "Unsupported method", "method", request.Method, "explanation",
-		"Caduceus only supports \"POST\" method.")
+		errorLog.Log(errorKey, "Unsupported method", "method", request.Method, messageKey,
+		"Caduceus only supports POST method.")
 		return
 	}
 
 	payload, err := ioutil.ReadAll(request.Body)
 	if err != nil {
 		response.WriteHeader(http.StatusBadRequest)
 		response.Write([]byte(fmt.Sprintf("Unable to retrieve the request body: %s.\n", err.Error)))
-		debugLog.Log(messageKey, "Unable to retrieve the request body.", logging.ErrorKey(), err.Error)
+		errorLog.Log(messageKey, "Unable to retrieve the request body.", errorKey, err.Error)
```

Comment:
@johnabass same as above.

Reply:
This log works fine, but code should not attempt to write anything to the response.  If a request body occurs, that indicates a low-level socket I/O error and the response/request pair should be abandoned.

Reply:
Understood. Removed lines 47-48.